src/sentry/lang/native/processing.py

```diff
@@ -455,12 +455,14 @@ def process_native_stacktraces(symbolicator: Symbolicator, data: Any) -> Any:
     return data
 
 
-def get_native_symbolication_function(data) -> Callable[[Symbolicator, Any], Any] | None:
+def get_native_symbolication_function(
+    data, stacktraces
+) -> Callable[[Symbolicator, Any], Any] | None:
```

Comment:
lets go all-in on typing and docs here :-)
```suggestion
def get_native_symbolication_function(
    data: Any,
    stacktraces: list[StacktraceInfo]
) -> Callable[[Symbolicator, Any], Any] | None:
    """
    Returns the appropriate symbolication function (or `None`) that will process
    the event, based on the Event `data`, and the supplied `stacktraces`.
    """
```